src/StatsdClient/Bufferize/StatsBufferize.cs

```diff
@@ -83,8 +83,7 @@ public bool OnIdle()
                 {
                     this._bufferBuilder.HandleBufferAndReset();
 
-                    // No need to wait as sending the value takes time.
-                    return false;
+                    return true;
```

Comment:
Should we just remove the return statement here, and just use the return statement below? Also, if this always returns true, should it then be void instead?

Reply:
I am going to check if `HandleBufferAndReset` can return true when it does something. As it requires more testing, I am going to todo it in another PR (or I am going to make this method void)